examples/docker/kafka-oauth-strimzi/kafka/oauth.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+usage() {
+  echo "Usage: $0 [USERNAME] [PASSWORD] [ARGUMENTS] ..."
+  echo
```

Comment:
I think you should add a sentence explaining what this command actually _does_. "Requests a refresh or access token from ...."